### ISSUE TITLE
Undetected unused import when module is in a subdirectory

### ISSUE THREAD

**Issue by aureliojargas** [PyCQA/pylint#3438]
### Steps to reproduce

Given this test file, that import two modules in a sub directory, but only one is used:

```console
$ cat nok.py 
import sub.one
import sub.two

sub.one.x = 1
$
```

The imported modules are empty in this example (their content is irrelevant):

```console
$ mkdir sub
$ touch sub/one.py
$ touch sub/two.py
```

### Current behavior

Pylint does not detect that `sub.two` is not used:

```console
$ pylint nok.py
************* Module nok
nok.py:1:0: C0114: Missing module docstring (missing-module-docstring)

-----------------------------------
Your code has been rated at 6.67/10
$
```

If those modules are not in a subdirectory, it works as expected (the unused module is detected):

```console
$ cat sub/ok.py 
import one
import two

one.x = 1

$ pylint sub/ok.py 
************* Module ok
sub/ok.py:1:0: C0114: Missing module docstring (missing-module-docstring)
sub/ok.py:2:0: W0611: Unused import two (unused-import)

-----------------------------------
Your code has been rated at 3.33/10
$
```

### Expected behavior

The unused `sub.two` module is detected.

### pylint --version output

    pylint 2.4.4
    astroid 2.3.3
    Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
    [GCC 8.3.0]

**Comment by dbaty** [comment]
Looks like a duplicate of #2583.

**Comment by PCManticore** [comment]
Thanks @dbaty
